olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/StreamLayerClient.h

```diff
@@ -162,6 +163,39 @@ class DATASERVICE_READ_API StreamLayerClient final {
    */
   client::CancellableFuture<UnsubscribeResponse> Unsubscribe();
 
+  /**
+   * @brief Downloads message data using data handle.
```

Comment:
```suggestion
   * @brief Downloads message data using a data handle from the given message.
```
Not entirely clear from where comes the mentioned data handle. Please consider adding more details.